.github/workflows/main.yml

```diff
@@ -25,9 +22,6 @@ jobs:
             TOXENV: py
 
         # PyPY
-        - python-version: pypy3.7
-          env:
-            TOXENV: pypy3
```

Comment:
The goal here is to keep a maximum and minimum version. So maybe switch to 3.8 isntead?